src/modules/core/components/InfoPopover/MemberInfoPopover/UserTokens.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import { BigNumberish } from 'ethers/utils';
+
+import Heading from '~core/Heading';
+import { getFormattedTokenValue } from '~utils/tokens';
+
+import styles from './MemberInfoPopover.css';
+import Numeral from '~core/Numeral';
+import { UserToken } from '~data/generated';
+
+const displayName = `InfoPopover.MemberInfoPopover.UserTokens`;
+
+interface Props {
+  totalBalance: BigNumberish;
+  nativeToken: UserToken;
+}
+
+const MSG = defineMessages({
+  labelText: {
+    id: 'InfoPopover.MemberInfoPopover.UserTokens.labelText',
+    defaultMessage: 'Tokens',
+  },
+});
+
+const UserTokens = ({ totalBalance, nativeToken }: Props) => {
+  const formattedTotalBalance = getFormattedTokenValue(
+    totalBalance,
+    nativeToken.decimals,
+  );
+
+  return (
+    <div className={styles.tokensContainer}>
+      <Heading
+        appearance={{
+          size: 'normal',
+          margin: 'none',
+          theme: 'grey',
+          weight: 'bold',
+        }}
```

Comment:
I mentioned this in the other PR as well:
- color is not properly applied _(theme `grey` still renders as `--dark`)_
- please check if the font size corresponds to the Figma specs